packages/dart_frog_cli/lib/src/daemon/domain/dev_server_domain.dart

```diff
@@ -216,4 +223,10 @@ class DevServerDomain extends DomainBase {
     }
     _devServerRunners.clear();
   }
+
+  bool _isValidHost(dynamic host) {
+    return host is String &&
+        RegExp(r'^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$')
+            .hasMatch(host);
+  }
 }
```

Comment:
**Nit:** 
Since the method doesn't need to be a member of the `DevServerDomain` we can consider it having it as a function instead.

```suggestion
}

bool _isValidHost(dynamic host) {
  return host is String &&
      RegExp(r'^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$')
          .hasMatch(host);
}
```

Reply:
Check [Commit](https://github.com/VeryGoodOpenSource/dart_frog/pull/924/commits/2ba0ff5f665d5c761d846e497377c5184204c536)